language/stdlib/modules/libra_configs.move

```diff
@@ -50,12 +50,11 @@ module LibraConfig {
     }
 
     // Publish a new config item to a new value and trigger a reconfiguration.
-    public fun publish_new_config<Config: copyable>(payload: Config) acquires Configuration {
+    public fun publish_new_config<Config: copyable>(payload: Config) {
         // TODO: impose proper permission checks
         // Callable by the any address for now.
 
         move_to_sender(T{ payload });
-        reconfigure_();
```

Comment:
Maybe some comments here? Publishing a new configuration will not trigger a reconfiguration immediately. Instead we need to wait all of the validators to start to use this config before triggering a reconfiguration change.